src/Main.java

```diff
@@ -1,7 +1,10 @@
 public class Main {
 
     public static void main(String[] args) {
-
+        System.out.println(sumFunction(8, 11));
+    }
+    public static int sumFunction (int n1, int n2) {
```

Comment:
Questa funzione dovrebbe chiamarsi sum

Reply:
io opterei per usare le convenzioni quindi definire i metodi tramite verbi, in questo caso "getSum", così da non fare confusione se vogliamo usare sum per definire una variabile o catturare quel valore .